test/functional/pdo_sqlsrv/pdo_ae_insert_pdoparam_nchar_size.phpt

```diff
@@ -0,0 +1,169 @@
+--TEST--
+Test for inserting encrypted data into nchar types columns with different sizes
+--DESCRIPTION--
+Test conversions between different nchar types of different sizes
+With or without Always Encrypted, implicit conversion works if:
+1. From input of PDO::PARAM_BOOL to a any nchar column
+2. From input of PDO::PARAM_INT to a any nchar column
+3. From input of PDO::PARAM_STR to a any nchar column
+4. From input of PDO::PARAM_LOB to a any nchar column
+--SKIPIF--
+<?php require('skipif_mid-refactor.inc'); ?>
+--FILE--
+<?php
+require_once("MsCommon_mid-refactor.inc");
+require_once("AEData.inc");
+
+$dataTypes = array("nchar", "nvarchar", "nvarchar(max)");
+$lengths = array(1, 8, 64, 512, 4000);
+
+try {
+    $conn = connect();
+    foreach ($dataTypes as $dataType) {
+        $maxcol = strpos($dataType, "(max)");
+        foreach ($lengths as $m) {
+            if ($maxcol !== false) {
+                $typeFull = $dataType;
+            } else {
+                $typeFull = "$dataType($m)";
+            }
+            echo "\nTesting $typeFull:\n";
+                
+            //create table containing nchar(m) or nvarchar(m) columns
+            $tbname = "test_" . str_replace(array('(', ')'), '', $dataType) . $m;
+            $colMetaArr = array(new ColumnMeta($typeFull, "c1"));
+            createTable($conn, $tbname, $colMetaArr);
```

Comment:
again, why only one column in this test?

Reply:
see explanation in the char test

Reply:
I can't seem to find it, but still it would be nice to put a comment to reference the other test and explain the similarity between the two tests